lib/web_ui/lib/src/engine/canvaskit/image.dart

```diff
@@ -11,29 +11,29 @@ import '../dom.dart';
 import '../html_image_codec.dart';
 import '../safe_browser_api.dart';
 import '../util.dart';
+import 'canvas.dart';
 import 'canvaskit_api.dart';
 import 'image_wasm_codecs.dart';
 import 'image_web_codecs.dart';
+import 'painting.dart';
+import 'picture.dart';
+import 'picture_recorder.dart';
 import 'skia_object_cache.dart';
 
 /// Instantiates a [ui.Codec] backed by an `SkAnimatedImage` from Skia.
-// TODO(yjbanov): Implement targetWidth and targetHeight support.
-//                https://github.com/flutter/flutter/issues/34075
 FutureOr<ui.Codec> skiaInstantiateImageCodec(Uint8List list,
     [int? targetWidth, int? targetHeight]) {
-  if (browserSupportsImageDecoder) {
+      // if we have either a target width or target height, use canvaskit to decode
```

Comment:
nit: Fix indentation. Make this into a full sentence with capitalization and full stop